examples/NeurIPS2019-Learn-to-Move-Challenge/README.md

```diff
@@ -1,8 +1,9 @@
- 
+# The Winning Solution for the NeurIPS 2019: Learn to Move Challenge
+
```

Comment:
Could you add an image demonstrating the performance? I will send you the gif.

Reply:
done